chromaui/chromatic-e2e
# [cypress]:add types for custom takeSnapshot command

Issue: #AP-4657

## What Changed


This PR adds types for the custom `takeSnapshot` Cypress command.

## How to test


1. Install the Canary in project.
2. Verify typings in VSCode.
